### ISSUE TITLE
Some panels (such as community plugins list) are unreadable

### ISSUE THREAD

**Issue by fncll** [dogwaddle/obsidian-gastown-theme.md#4]
When I look at the list of community plugins, the information is unreadable except for when I hover over an item (see screenshot). I've had this issue with a few themes...perhaps something has changed or been added to Obsidian?

<img width="976" alt="CleanShot 2021-06-27 at 12 30 03@2x" src="https://user-images.githubusercontent.com/1208889/123557067-ab33a000-d743-11eb-82fe-c096ac4e5367.png">

**Comment by dogwaddle** [comment]
@fncll yes, they need to rework how their community themes page works, such as allowing users to remove themes that are no longer maintained. Because the app is still in active development, a lot of the themes get broken. In the case of this theme, I stopped actively fixing it a long time ago because it is not the theme that I use daily in my own vault. I only maintain the primary theme that I use.
